src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout.iOS.cs

```diff
@@ -11,11 +11,13 @@
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
 using UIKit;
+using Uno.UI;
 
 namespace Windows.UI.Xaml.Controls
 {
 	public partial class MenuFlyout
 	{
+		private static DependencyProperty OverrideIosCancelTextProperty = ToolkitHelper.GetProperty("Uno.UI.Toolkit.MenuFlyoutExtensions", "OverrideIosCancelText");
```

Comment:
Discoverability will likely be easier if the target intent is first in the property name:

```suggestion
		private static DependencyProperty CancelTextIosOverrideProperty = ToolkitHelper.GetProperty("Uno.UI.Toolkit.MenuFlyoutExtensions", "CancelTextIosOverride");
```